src/components/Main/Main.tsx

```diff
@@ -114,6 +114,10 @@ function Main() {
   )
 
   useEffect(() => {
+    if (autorunSimulation) {
+      debouncedRun(allParams, scenarioState, severity)
+    }
+
```

Comment:
Runs the simulation if autorun changed to true, regardless of whether parameters in URL location have changed.